packages/scripts.nix

```diff
@@ -16,6 +16,83 @@
     text = builtins.readFile ./destroy-coco-aks.sh;
   };
 
+  upload-image = writeShellApplication {
+    name = "upload-image";
+    runtimeInputs = with pkgs; [
+      azure-cli
+      gnused
+      uplosi
+    ];
+    text =
+      let
+        image = pkgs.image-podvm;
+      in
+      ''
+        set -euo pipefail
+
+        subscriptionId=""
+        location="GermanyWestCentral"
+        resourceGroup=""
+
+        for i in "$@"; do
+          case $i in
+          --subscription-id=*)
+            subscriptionId="''${i#*=}"
+            shift
+            ;;
+          --location=*)
+            location="''${i#*=}"
+            shift
+            ;;
+          --resource-group=*)
+            resourceGroup="''${i#*=}"
+            shift
+            ;;
+          *)
+            echo "Unknown option $i"
+            exit 1
+            ;;
+          esac
+        done
+
+        set -x
```

Comment:
Debugging leftover or intended?

Reply:
Neither, I copied parts of the script from `create-coco-aks.sh` and that line was already present there (it's also in `destroy-coco-aks.sh`). I'm not invested in having this, we can keep it or remove it, I'd be fine either way.